spec/spec_helper.rb

```diff
@@ -1,17 +1,16 @@
+# frozen_string_literal: true
+
 require 'selenium-webdriver'
 require 'webdrivers'
 require 'rspec'
 
 RSpec.configure do |config|
-
-	config.before(:each) do
-		@driver = Selenium::WebDriver.for :chrome
-		@driver.manage.window.maximize
-    	@driver.get("https://www.abv.bg/")
-	end
-
-	config.after(:each) do
-		@driver.quit
-	end
-
-end
\ No newline at end of file
+  config.before(:each) do
+    @driver = Selenium::WebDriver.for :chrome
+    @driver.manage.window.maximize
+    @driver.get('https://www.abv.bg/')
```

Comment:
it is a nice practice to store urls in a separate file with other similar variables that are used in the project